amx/server/syscalls.lua

```diff
@@ -2763,15 +2763,37 @@ function SetPlayerChatBubble(amx, player, text, color, dist, exptime)
 end
 
 
--- In MTA we can't set the type of the request. 
--- When we add 'data' it becomes POST request.
--- TODO: arguments "index" and "type" here only for compatibility.
+-- Now we have all of RESTful types of requests. Our function is better!
+-- The SAMP documentation said about 'url' - "The URL you want to request. (Without 'http://')"

```

Comment:
Does this mean that the default scripts won't work? If so, I suggest that:

- If string starts with a protocol `SOMESTRING://`, don't change the string
- If string is missing a protocol, prefix `http://`

This supports original code + if SA-MP adds support like this later, it most likely will be done this way

Reply:
> Does this mean that the default scripts won't work?

I always test all functions in SAMP and in our AMX. It fully compatible in SAMP.

Reply:
Cool! I didn't know that :)

(Testing with `{ maxRedirects = 0 }` it looks like it always does HTTP. But it obviously follows redirects to HTTPS with the default.)